### PR TITLE
ITSASU-273: Migrate property businesses started page

### PR DESCRIPTION
**Pull request by llmikeyj**

### CODE REVIEW

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/PropertyTradingAfter.scala.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+@(propertyTradingBeforeDateForm: Form[YesNo], postAction: Call, startDateLimit: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@inputContent = {
+    <p class="govuk-body" id="property-trading-hint">@messages("property_trading_after.line")</p>
+}
+
+@ariaDescribedBy = {
+    @if(propertyTradingBeforeDateForm("yes-no").error.isDefined) {aria-describedby="yes-no-error"}
+}
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/PropertyTradingAfter.scala.html:41]: Looks like this is no longer used, could you remove this?

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/PropertyTradingAfter.scala.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    @form(action = postAction) {
+
+            @radioHelper(
+                field = propertyTradingBeforeDateForm(fieldName),
+                isPageHeading = false,
+                inline = true,
+                radioItems = Seq(
+                    RadioItem(value = Some(option_yes), content = HtmlContent(messages("base.yes"))),
+                    RadioItem(value = Some(option_no), content = HtmlContent(messages("base.no")))),
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/PropertyTradingAfter.scala.html:57]: Don't need to be HtmlContent, Text should be used instead

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/PropertyTradingAfter.scala.html (hunk 1 above)
Comment (llmikeyj) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/PropertyTradingAfter.scala.html:41]: Okay

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/PropertyTradingAfter.scala.html (hunk 2 above)
Comment (llmikeyj) [reply app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/PropertyTradingAfter.scala.html:57]: Okay

**Pull request merged**